EFCore-Adapter.UnitTest/EFCore-Adapter.UnitTest.csproj

```diff
@@ -8,8 +8,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="2.9.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.4" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0-preview.4.20251.6" />

```

Comment:
I think 5.0 is not released yet, we can use a stable version here.

Reply:
The stable version will throw a warning like this:
![image](https://user-images.githubusercontent.com/42855245/83960387-d0e0d380-a8ba-11ea-855a-39910ab38352.png)
But it looks not affect the result.

Reply:
Ok, I think use 5.0 is fine.